ppr-ui/src/composables/address/BaseAddress.vue

```diff
@@ -210,7 +208,10 @@ export default defineComponent({
         schemaLocal.value.region = origRegionRules
       } else {
         schemaLocal.value.postalCode = origPostalCodeRules.concat([baseRules.maxLength(15)])
-        schemaLocal.value.region = [baseRules.maxLength(2), ...spaceRules]
+        for (let i = 0; i < schemaLocal.value.region.length; i++) {
```

Comment:
hey steph, this should be `schemaLocal.value.region.length - 1` I think? Might not matter depending on how the array pop() works on an empty list

Reply:
if the length is 3, it's going to pop 0, 1, and 2 , which are 3 entries.

I did discover that the concat wasn't behaving nicely though, so I made a change there

Reply:
Oh right yeah totally

Reply:
I guess if it works it works. Maybe since it does the 'pops' it triggers the change this way